python_modules/dagster/dagster/core/definitions/decorators/repository.py

```diff
@@ -66,7 +65,7 @@ def __call__(self, fn: Callable[[], Any]) -> RepositoryDefinition:
                 raise DagsterInvalidDefinitionError(
                     "Bad return value from repository construction function: all elements of list "
                     "must be of type JobDefinition, GraphDefinition, PipelineDefinition, "
-                    "PartitionSetDefinition, ScheduleDefinition, SensorDefinition, or ForeignAsset. "
+                    "PartitionSetDefinition, ScheduleDefinition, or SensorDefinition. "
```

Comment:
nit: can we add AssetCollection to this list? Probably should have been in my previous PR, but better late than never

Reply:
Yes